src/QuartzNET-DynamoDB/JobStore.cs

```diff
@@ -1458,6 +1471,14 @@ public virtual string InstanceName
         /// </summary>
         private void CreateOrUpdateCurrentSchedulerInstance()
         {
+            if (!_periodicExecutionTrackers["CreateOrUpdateCurrentSchedulerInstance"].ShouldExecute())
```

Comment:
It just occurred to me that this would have been a nice place to use aspect-oriented programming. This code could have been in an attribute.